demonstrations/tutorial_kernel_module.py

```diff
@@ -0,0 +1,596 @@
+r"""Quantum Embedding Kernels with the kernels module
+=====================================================
+
+.. meta::
+    :property="og:description": Kernels and alignment training with Pennylane.
+    :property="og:image": https://pennylane.ai/qml/_images/QEK_thumbnail.png
+
+.. related::
+
+    tutorial_kernel_based_training Kernel-based training with scikit-learn
+    tutorial_data_reuploading_classifier Classification with data reuploading
+
+*Authors: Peter-Jan Derks, Paul Fährmann, Elies Gil-Fuster, Tom
+Hubregtsen, Johannes Jakob Meyer and David Wierichs. Posted: 24 June 2021*
+
+Kernel methods are one of the cornerstones of classical machine
+learning. To understand what a kernel method does, let’s first revisit
+one of the simplest methods to assign binary labels to datapoints:
+linear classification.
```

Comment:
I would tell a reader more what to expect here...the header may help advanced users to jump to the next section.

```suggestion
Kernel methods are one of the cornerstones of classical machine
learning. This tutorial will introduce the basic functionality of PennyLane's  
[kernels module](https://pennylane.readthedocs.io/en/latest/code/qml_kernels.html). 

What are kernel methods?
-----------------------------------

To understand what a kernel method does, let’s first revisit
one of the simplest methods to assign binary labels to datapoints:
linear classification.
```

Reply:
Agree, was just about to suggest this 💯

Reply:
In terms of the summary/intro, could add

- an additional sentence inbetween the two, making note of the *quantum* nature of the tutorial
- extend the final sentence, so that a reader knows what the demo will achieve
- the paper this is based on.

```markdown
Kernel methods are one of the cornerstones of classical machine
learning. Something something extends to quantum.
This tutorial will introduce the basic functionality of PennyLane's  
[kernels module](https://pennylane.readthedocs.io/en/latest/code/qml_kernels.html)
by exploring...
```